aries_cloudagent/wallet/askar.py

```diff
@@ -582,8 +582,10 @@ async def set_did_endpoint(
 
         """
         did_info = await self.get_local_did(did)
-        if did_info.method != SOV:
-            raise WalletError("Setting DID endpoint is only allowed for did:sov DIDs")
+        if did_info.method != SOV and did_info.method != INDY:
+            raise WalletError(
+                "Setting DID endpoint is only allowed for did:sov or did:indy DIDs"
+            )
```

Comment:
Setting DID Endpoint in this way will get picked up by the legacy resolution support in Indy VDR but doesn't get picked up by the universal resolver, just as an aside. Ideally, we set this by updating the nym with diddocContent rather than with an attrib. We can address that in the future, though, of course.